spec/cucumber/formatter/ansicolor_spec.rb

```diff
@@ -29,6 +29,16 @@ module Formatter
 
         expect(passed('foo')).to eq 'foo'
       end
+
+      it 'works with custom colors' do
+        apply_custom_colors('passed=red,bold')
+        expect(passed('foo')).to eq "\e[31m\e[1mfoo\e[0m\e[0m"
+        apply_custom_colors('passed=green')
```

Comment:
what is this line doing?

Reply:
It reset the colors.
Without this, other tests executed after that one may have the custom color kept to 'passed=red,bold'

Reply:
Place that inside an after hook inside the relevant scope. Maybe inside a nicely named method like `reset_colours_to_default` or something